mono/metadata/assembly-load-context.c

```diff
@@ -1,13 +1,20 @@
 #include "config.h"
+#include "mono/metadata/assembly.h"
 #include "mono/metadata/domain-internals.h"
 #include "mono/metadata/icall-decl.h"
 #include "mono/metadata/loader-internals.h"
 #include "mono/metadata/loaded-images-internals.h"
 #include "mono/utils/mono-error-internals.h"
+#include "mono/utils/mono-logger-internals.h"
```

Comment:
`#include` possibly under `#if` but there are reasons why not: sorting or don't-care or order-dependent.